### ISSUE TITLE
Jotunn 2.13 incompatiblilty

### ISSUE THREAD

**Issue by MSchmoecker** [bonesbro/ValFloorsAreRoofs#1]
After the Jotunn 2.13 update, the mod throws errors when loaded before JVL.

FloorsAreRoofs is missing the attribute `[BepInDependency(Jotunn.Main.ModGuid)]` next to the BepInPlug in order to tell BepInEx that it should be loaded after Jotunn. Otherwise unexpected issues like this can occur, since the load order is not reliable without setting it explicitly.

```
[Info   :   BepInEx] Loading [Floors Are Roofs 2.0]
[Info   : Unity Log] [floorsareroofs]: Initializing using config settings: PrefabsToChange:[wood_floor,wood_floor_1x1], Hammers:[Hammer], RemoveRainDamage:[True]
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Jotunn.Logger.Log (BepInEx.Logging.LogLevel level, BepInEx.BepInPlugin sourceMod, System.Object data) (at D:/Projects/Jotunn/JotunnLib/Logger.cs:83)
Jotunn.Logger.LogInfo (System.Object data) (at D:/Projects/Jotunn/JotunnLib/Logger.cs:146)
Jotunn.Managers.PieceManager.Jotunn.IManager.Init () (at D:/Projects/Jotunn/JotunnLib/Managers/PieceManager.cs:94)
Jotunn.Managers.PieceManager..cctor () (at D:/Projects/Jotunn/JotunnLib/Managers/PieceManager.cs:37)
Rethrow as TypeInitializationException: The type initializer for 'Jotunn.Managers.PieceManager' threw an exception.
FloorsAreRoofs.FloorsAreRoofsPlugin.Awake () (at <a3f6643ed373433eb1319e7a1ba2ea79>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.UI.Image:OnCanvasHierarchyChanged()
```

**Comment by bonesbro** [comment]
Thank you!

**Comment by bonesbro** [comment]
Addressed with version 2.0.1, commit https://github.com/bonesbro/ValFloorsAreRoofs/commit/2731557d2d2b9a439022f305fe1b65aa7280efd9.

**Comment by bonesbro** [comment]
Released as 2.0.2 on https://valheim.thunderstore.io/package/bonesbro/FloorsAreRoofs/ and https://www.nexusmods.com/valheim/mods/1039.
